packages/imperative/src/config/src/ProfileInfo.ts

```diff
@@ -772,14 +777,27 @@ export class ProfileInfo {
      * detected the existence of old-school V1 profiles. We will not work with the
      * V1 profiles. This function can let you tell a user that they are incorrectly
      * trying to use V1 profiles.
-     *
+     * @deprecated Use non-static method instead to detect V2 profiles
      * @returns True - Means there is *NO* team config *AND* we detected that a V1 profile exists.
      *          False otherwise.
```

Comment:
I agree with the new deprecation message, but since it no longer checks for existence of v2 profiles, should we update the `@returns` info in typedoc?

```suggestion
     * @returns {boolean} `true` if a V1 profile exists, and `false` otherwise.
```

Reply:
I think that the original static method always tested for the existence of a team config. That test occurs in configUtils.onlyV1ProfilesExist. I thought that the reason for the new member function was to accommodate some imperative initialization actions which would not occur if the static method is called before the ProfileInfo class is instantiated.

With my level of understanding, it does raise the question of why we need to make an additional test for the existence of a team config in the new member function?

Reply:
Ah, I see, thanks for clarifying on that Gene. That's actually why I made the request for changes as I noticed that the new method was doing a check for team config, so I assumed that the `ConfigUtils.onlyV1ProfilesExist` function did not. 

Since it does check for the team config in `ImperativeConfig.instance`, my request has changed to a follow-up question for my own clarification: is the team config check in the new method necessary?

Reply:
> Could we somehow assign ImperativeConfig.instance.config from Zowe Explorer and eliminate the need for the first condition?

Yes, that approach would also work although I felt it might be a bit hacky, and I wouldn't expect other SDK users to know they need to set `ImperativeConfig.instance.config` in order for the getter to work.

Good catch on the Typedoc message, I'll update it 😋

Reply:
Here is the way that I read this logic: The expression "ImperativeConfig.instance.config?.exists" in ConfigUtils can be false if there is no config or if there **IS** a config, but ImperativeConfig has not been fully initialized. The "extra" test in ProfileInfo avoids getting a false negative result in ConfigUtils, by ensuring that ConfigUtils is only called if there really is no config file. Please correct me if I have misinterpreted. It took me a long time re-reading this logic to draw my conclusion. The logic is not intuitively obvious, but I get that it is quite necessary.